recipes-security/fail2ban/python3-fail2ban_0.11.2.bbappend

```diff
@@ -0,0 +1,8 @@
+PR="r1"
+
+# fail2ban has an explicit dependency on busybox-syslog
+# which, in turn, conflicts with rsyslog. 
+# we'd have to forcibly remove it or the newer gwc core
+# won't be installable.
+VIRTUAL-RUNTIME_syslog = ""
+VIRTUAL-RUNTIME_base-utils-syslog = ""
```

Comment:
I beleive the first one (`VIRTUAL-RUNTIME_syslog`) is not really used so should be removed.

For the second one I see two things:
1. It should be `VIRTUAL-RUNTIME_base-utils-syslog = "rsyslog"` so it "announces" that we are using `rsyslog`. As you put it, it would remove it totally and would seem from the variable we don't have a logger.
2. I think it would be much better instead of putting it here, to put it in the configuration of our system, so also other packages (if any and in case for the future) know that we are using `rsyslog` as the logger. So I would put at the end of the file `conf/machine/imx8xq-smarcore.conf` the lines:
```
# Make rsyslog our default logger
VIRTUAL-RUNTIME_base-utils-syslog = "rsyslog"
```

I've tested a rebuild like that and is then fine (without it would, as you mention, complain about the conflict)